gapic/doc.go

```diff
@@ -59,7 +59,7 @@
 //
 // Use of Context
 //
-// The ctx passed to NewClient is used for authentication requests and
+// The ctx passed to NewRegistryClient is used for authentication requests and
```

Comment:
This is generated code, and this change gets clobbered when we rerun ./tools/GENERATE-GAPIC.sh. I think it's an upstream problem in the generator.

Reply:
Should we merge and file a new issue on this?

Reply:
rolled back the doc change